core/src/main/java/de/danoeh/antennapod/core/preferences/GpodnetPreferences.java

```diff
@@ -82,19 +92,25 @@ private static synchronized void ensurePreferencesLoaded() {
     private static void writePreference(String key, String value) {
         SharedPreferences.Editor editor = getPreferences().edit();
         editor.putString(key, value);
-        editor.commit();
+        editor.apply();
```

Comment:
Since we always keep all values as member variables (and access them that way), there's really no need to perform the write operations synchronously